.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: cpp
+compiler: gcc
```

Comment:
The problem with using the Travis `cpp` language instead of the examples I linked you before is Travis comes with a very old GCC version which could cause compilation errors when you are using C++11 features, see also:

https://docs.travis-ci.com/user/languages/cpp/#C11-C%2B%2B11-(and-Beyond)-and-Toolchain-Versioning

That's why the Travis configs I linked you were installing an up-to-date compiler first.